app/javascript/components/footer/footer.scss

```diff
@@ -0,0 +1,131 @@
+ @import '../.././styles/main.scss';
+
+.content {
+  grid-row-gap: 25px;
+  &__desktop {
+    @media only screen and (min-width: 700px) {
+      display: flex;
+      justify-content: space-between;
+      padding: 0 28px;
+      width: 100%;
+   }
+  }
+}
+
+.footer__logo {
+width: 234px;
+height: 51px;
+}
```

Comment:
check the indentation here, and sort alphabetically

Reply:
done